apps/bfd-model/bfd-model-rif-samples/src/test/java/gov/cms/bfd/model/rif/samples/SampleDataColumnsTest.java

```diff
@@ -86,18 +85,18 @@ private void verifyColumns(StaticRifResourceGroup sampleGroup) {
        * Loop through the columns in the sample data and ensure that
        * our column enums match.
        */
-      for (int col = 0; col < columnsInSample.length; col++) {
-        String columnNameFromEnum = columnsInEnum[col].name();
-        String columnNameFromSample = columnsInSample[col];
-        assertEquals(
-            columnNameFromSample,
-            columnNameFromEnum,
-            String.format(
-                "Unable to match column '%d' from sample data for '%s'.\nSample Columns: %s\nEnum Columns:   %s\n",
-                col,
-                sampleFile.name(),
-                toHeaderFormat(columnsInSample, c -> c),
-                toHeaderFormat(columnsInEnum, c -> c.name())));
+
+      HashSet<String> sampleSet = new HashSet<>(Arrays.asList(columnsInSample));
```

Comment:
Changes here are to prevent the order from mattering. Instead of checking that the columns have the same index, we only care that all columns are included.